Source/relay/TourGuide/Items of the Month/2022/Tiny Stillsuit.ash

```diff
@@ -2,78 +2,89 @@
-	description.listAppend("Two gross tastes that taste horrible together.");
-	//an amish paradise is as primitive as can be
-	int sweatCalcSweat;
-	int sweatCalcAdvs;
+  int sweatAdvs = (fam_sweat_o_meter ** 0.4);
+  int sweatAdvsConversion = (sweatAdvs - 0.5) ** 2.5;
+  int nextSweatDrams = (sweatAdvs+0.51) ** 2.5; # - fam_sweat_o_meter;
+  
+  string title;
+  string [int] description;
+  string url = "inventory.php?action=distill&pwd=" + my_hash();
+  title = HTMLGenerateSpanFont(fam_sweat_o_meter + " drams of stillsuit sweat", "purple");
+  description.listAppend("Two gross tastes that taste horrible together.");
+  //an amish paradise is as primitive as can be
+  int sweatCalcSweat;
+  int sweatCalcAdvs;
 
-	if (fam_sweat_o_meter >= 279) {
-		sweatCalcSweat = 358;
-		sweatCalcAdvs = 10;
-	}
-	else if (fam_sweat_o_meter >= 211) {
-		sweatCalcSweat = 279;
-		sweatCalcAdvs = 9;
-	}
-	else if (fam_sweat_o_meter >= 155) {
-		sweatCalcSweat = 211;
-		sweatCalcAdvs = 8;
-	}
-	else if (fam_sweat_o_meter >= 108) {
-		sweatCalcSweat = 155;
-		sweatCalcAdvs = 7;
-	}
-	else if (fam_sweat_o_meter >= 71) {
-		sweatCalcSweat = 108;
-		sweatCalcAdvs = 6;
-	}
-	else if (fam_sweat_o_meter >= 43) {
-		sweatCalcSweat = 71;
-		sweatCalcAdvs = 5;
-	}
-	else if (fam_sweat_o_meter >= 23) {
-		sweatCalcSweat = 43;
-		sweatCalcAdvs = 4;
-	}
-	else if (fam_sweat_o_meter >= 10) {
-		sweatCalcSweat = 23;
-		sweatCalcAdvs = 3;
-	}
-	
-	if (fam_sweat_o_meter > 358) {
-		description.listAppend("" + HTMLGenerateSpanOfClass("11", "r_bold") + " advs when guzzling now (costs 1 liver).");
-		description.listAppend("You should probably guzzle your sweat now.");
-	}
-	else {
-		description.listAppend("" + HTMLGenerateSpanOfClass(sweatCalcAdvs, "r_bold") + " advs when guzzling now (costs 1 liver).");
-		description.listAppend("" + HTMLGenerateSpanOfClass(sweatCalcSweat - fam_sweat_o_meter, "r_bold") + " more sweat until +1 more adventure. (" + CEIL(sweatCalcSweat - fam_sweat_o_meter)/3 + " combats on current familiar)");
-	}
-		
-#	description.listAppend("" + HTMLGenerateSpanOfClass(sweatAdvs, "r_bold") + " advs when guzzling now (costs 1 liver).");
+  if (fam_sweat_o_meter >= 279)
+  {
+    sweatCalcSweat = 358;
+    sweatCalcAdvs = 10;
+  }
+  else if (fam_sweat_o_meter >= 211)
+  {
+    sweatCalcSweat = 279;
+    sweatCalcAdvs = 9;
+  }
+  else if (fam_sweat_o_meter >= 155)
+  {
+    sweatCalcSweat = 211;
+    sweatCalcAdvs = 8;
+  }
+  else if (fam_sweat_o_meter >= 108)
+  {
+    sweatCalcSweat = 155;
+    sweatCalcAdvs = 7;
+  }
+  else if (fam_sweat_o_meter >= 71)
+  {
+    sweatCalcSweat = 108;
+    sweatCalcAdvs = 6;
+  }
+  else if (fam_sweat_o_meter >= 43)
+  {
+    sweatCalcSweat = 71;
+    sweatCalcAdvs = 5;
+  }
+  else if (fam_sweat_o_meter >= 23)
+  {
+    sweatCalcSweat = 43;
+    sweatCalcAdvs = 4;
+  }
+  else if (fam_sweat_o_meter >= 10)
+  {
+    sweatCalcSweat = 23;
+    sweatCalcAdvs = 3;
+  }
+  
+  if (fam_sweat_o_meter > 358 && __misc_state["in run"]) {
```

Comment:
the only thing i'd note here is that we don't have adv totals in the messy if/else above beyond 10 advs, so the final else here is going to mislead anyone using this tile in aftercore. might want to just remove the "you should probably guzzle your sweat now" line and replace it with something along the lines of "You will get > 10 adventures by guzzling sweat now." to handle the fact that tes could not figure out the root function stuff and there appear to be weird ash bugs with it.

Reply:
Fixed in latest. Thanks to @horrible-little-slime for his maths help as my brain is currently undergoing maintenance.